microcosm_flask/conventions/swagger.py

```diff
@@ -50,7 +48,7 @@ def configure_discover(self, ns, definition):
         """
         @self.graph.route(ns.singleton_path, Operation.Discover, ns)
         def discover():
-            swagger = build_swagger(self.graph, ns.version, ns.path, self.operations)
```

Comment:
First main change: the swagger layer had its own path calculation; it now delegates to the `Namespace` or the `make_path()` function from the routing layer